lib/string_master/string_master.rb

```diff
@@ -67,10 +67,17 @@ def urls_to_images(options = {})
   def urls_to_links(options = {})
     wrap_with = options[:wrap_with] || ['','']
     html_options = options[:html_options] || ''
-    @modified_string.gsub!(
-      /(\s|^|\A|\n|\t|\r)((http|https):\/\/.*?)([,.])?(\s|$|\n|\Z|\t|\r|<)/,
-      '\1<a href="\2" ' + html_options + '>\2</a>\4\5'
-    )
+    validate = options[:ascii_only] || false
+
+    # ascii_only option - show URLs with warning about contain non-latin characters
+    if validate == true
```

Comment:
Why use an additional variable `validate` at all? Why not `if options[:ascii_only]`?

Reply:
Oops. Fixed